### PR TITLE
New cheat sheet: Desktop Environments

### PR DESCRIPTION
**Pull request by laorynas**
## People to notify

@laorynas

---

Instant Answer Page: https://duck.co/ia/view/desktop_environments

### CODE REVIEW

**Comment by daxtheduck** [comment]
@laorynas Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/desktop-environments.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{
+    "id": "desktop_environments_cheat_sheet",
+    "name": "Desktop Environments",
+    "description": "Desktop Environments is a list of Desktop Environments and their descriptions",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/desktop-environments.json]: Change this to `A desktop environment provides a complete graphical user interface (GUI) for a system`

Comment (laorynas) [inline share/goodie/cheat_sheets/json/desktop-environments.json]: I will change the description soon

**Comment by gautamkrishnar** [comment]
CC: #3821

**Comment by mintsoft** [comment]
@laorynas @gautamkrishnar I've repushed that build and it's now passed; so I'm chalking this up to some network connectivity style issue

**Pull request merged**